content/zh-cn/docs/reference/access-authn-authz/node.md

```diff
@@ -13,193 +12,215 @@ reviewers:
-  [certificationsigningrequests API](/zh/docs/reference/access-authn-authz/certificate-signing-requests/)
-  的读/写权限
-* 为委派的身份验证/授权检查创建 TokenReview 和 SubjectAccessReview 的能力
+鉴权相关操作：
 
 <!--
 * read/write access to the [CertificateSigningRequests API](/docs/reference/access-authn-authz/certificate-signing-requests/) for TLS bootstrapping
 * the ability to create TokenReviews and SubjectAccessReviews for delegated authentication/authorization checks
 -->
+* 对于基于 TLS 的启动引导过程时使用的
+  [certificationsigningrequests API](/zh-cn/docs/reference/access-authn-authz/certificate-signing-requests/)
+  的读/写权限
+* 为委派的身份验证/授权检查创建 TokenReview 和 SubjectAccessReview 的能力
 
-在将来的版本中，节点鉴权器可能会添加或删除权限，以确保 kubelet 具有正确操作所需的最小权限集。
 <!--
 In future releases, the node authorizer may add or remove permissions to ensure kubelets
 have the minimal set of permissions required to operate correctly.
 -->
+在将来的版本中，节点鉴权器可能会添加或删除权限，以确保 kubelet 具有正确操作所需的最小权限集。
 
-为了获得节点鉴权器的授权，kubelet 必须使用一个凭证以表示它在 `system:nodes` 组中，用户名为 `system:node:<nodeName>`。
-上述的组名和用户名格式要与 [kubelet TLS 启动引导](/zh/docs/reference/access-authn-authz/kubelet-tls-bootstrapping/)过程中为每个 kubelet 创建的标识相匹配。
 <!--
 In order to be authorized by the Node authorizer, kubelets must use a credential that identifies them as
 being in the `system:nodes` group, with a username of `system:node:<nodeName>`.
 This group and user name format match the identity created for each kubelet as part of
 [kubelet TLS bootstrapping](/docs/reference/access-authn-authz/kubelet-tls-bootstrapping/).
 -->
+为了获得节点鉴权器的授权，kubelet 必须使用一个凭证以表示它在 `system:nodes`
+组中，用户名为 `system:node:<nodeName>`。上述的组名和用户名格式要与
+[kubelet TLS 启动引导](/zh-cn/docs/reference/access-authn-authz/kubelet-tls-bootstrapping/)
+过程中为每个 kubelet 创建的标识相匹配。
+
+<!--
+The value of `<nodeName>` **must** match precisely the name of the node as registered by the kubelet. By default, this is the host name as provided by `hostname`, or overridden via the [kubelet option](/docs/reference/command-line-tools-reference/kubelet/) `--hostname-override`. However, when using the `--cloud-provider` kubelet option, the specific hostname may be determined by the cloud provider, ignoring the local `hostname` and the `--hostname-override` option. 
+For specifics about how the kubelet determines the hostname, see the [kubelet options reference](/docs/reference/command-line-tools-reference/kubelet/).
+-->
+`<nodeName>` 的值**必须**与 kubelet 注册的节点名称精确匹配。默认情况下，这是由
+`hostname` 提供的主机名，或者通过 kubelet `--hostname-override`
+[选项](/zh-cn/docs/reference/command-line-tools-reference/kubelet/) 覆盖。
+但是，当使用 `--cloud-provider` kubelet 选项时，具体的主机名可能由云提供商确定，
+忽略本地的 `hostname` 和 `--hostname-override` 选项。有关
+kubelet 如何确定主机名的详细信息，请参阅
+[kubelet 选项参考](/zh-cn/docs/reference/command-line-tools-reference/kubelet/)。
 
-要启用节点授权器，请使用 `--authorization-mode = Node` 启动 apiserver。
 <!--
 To enable the Node authorizer, start the apiserver with `--authorization-mode=Node`.
 -->
+要启用节点授权器，请使用 `--authorization-mode = Node` 启动 API 服务器。
```

Comment:
```suggestion
要启用节点鉴权器，请使用 `--authorization-mode = Node` 启动 API 服务器。
```

Reply:
I have no preference over 鑑權 or 授權, and it's not listed in [common terms](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#%E6%9C%AF%E8%AF%AD%E5%AF%B9%E7%85%A7).

How do you think the file should be changed?

Reply:
The [changes](https://github.com/kubernetes/website/pull/35552/files) have 13 `鉴权器` and 1 `授权器` for the same word `authorizer`.

As a logical chain, if we use `鉴权器` for authorizer, then whether authorize and authorization should be `鉴权`？This may include many changes, not suggested.

Reply:
'=' 前后的空格是哪里来的